composer.json

```diff
@@ -31,9 +28,11 @@
         "phpunit/phpunit": "^9.5",
         "symfony/phpunit-bridge": "^6.0"
     },
+    "conflict": {
+        "guzzlehttp/psr7": "<2.0.0"
```

Comment:
We don't have direct dependency to `guzzlehttp/psr7` but without this hack `--prefer-lowest` installs 1.4.0 which doesn't have `psr/http-factory` and official transport is not able to establish connection (see  https://github.com/elastic/elastic-transport-php/blob/main/src/NodePool/Node.php#L33)

Reply:
I wonder if there is a way we could add a note around this to the code somewhere. I'm worried someone comes along eventually and removes it and we don't see the side effects immediately. The good news is, that now if someone goes back to where it was introduced, they see these comments.